steps/01/README.md

```diff
@@ -61,9 +61,10 @@ The manifest serves as a crucial configuration file for applications, components
 
 Let's start by creating a new file named `manifest.json` in the webapp folder and define its essential attributes:
 
--   The `_version` attribute in the root is mandatory in the manifest. It indicates the format version of the manifest, which is crucial for identifying application settings whenever the file is read by various tools. As new features or changes are introduced in future versions, maintaining the correct version number ensures compatibility and precise interpretation of the manifest's contents. 
+-   The `_version` attribute is a mandatory field in the app descriptor that indicates the format version of the descriptor. This attribute is crucial for identifying application settings when the descriptor is read by various tools. As new features or changes are introduced in future versions of the descriptor, the version number helps ensure compatibility and proper interpretation of the descriptor's contents. Consequently, with each new version of OpenUI5 a corresponding version of the app descriptor is released. For this tutorial, we have determined that our app requires a minimum OpenUI5 version of 1.132. Therefore, we specify the descriptor format version as 1.68.0, aligning it with the appropriate OpenUI5 version.
```

Comment:
You are still using the outdated term "app descriptor" here. I would recommend to keep the original description instead. Also, you mention here that with 1.132 we need to specify the manifest version 1.68.0 but in fact in the below json code snippet (as well as in the sure files) it says 1.60.

Reply:
Good catch! Thx Margot. I have reset the readme in step 01 to the current version. The initial update on this file is already covered by your PR few weeks back.